vmm/src/vm.rs

```diff
@@ -2173,6 +2176,11 @@ impl Vm {
             self.vm.tdx_finalize().map_err(Error::FinalizeTdx)?;
         }
 
+        // Unfreeze the vm for MSHV
+        #[cfg(feature = "mshv")]
+        if current_state == VmState::Created {
+            self.vm.resume().map_err(Error::UnfreezeVm)?;
```

Comment:
Should this be ` MigratableError::Resume` ?

Reply:
boot() does not return MigratableError so we can't return.